xyz-hub-service/src/main/java/com/here/xyz/hub/connectors/RpcClient.java

```diff
@@ -324,22 +324,24 @@ private void parseResponse(final Marker marker, byte[] bytes, final Handler<Asyn
         parseResponse(marker, payload, callback);
       }
     } catch (NullPointerException e) {
-      logger.error(marker, "Received empty response, but expected a JSON response.", e);
+      logger.error(marker, "Received empty response from connector \"{}\", but expected a JSON response.", getConnector().id, e);
       callback.handle(Future.failedFuture(new HttpException(BAD_GATEWAY, "Received an empty response from the connector.")));
     } catch (JsonMappingException e) {
-      logger.error(marker, "Error in the provided content {}", stringResponse, e);
+      logger.error(marker, "Error in the provided content {} from connector \"{}\"", stringResponse, getConnector().id, e);
       HttpException parsedError = getErrorMessage(stringResponse);
       callback.handle(Future.failedFuture(parsedError != null ? parsedError : new HttpException(BAD_GATEWAY,
-          "Invalid content provided by the connector: Invalid JSON type. Expected is a sub-type of XyzResponse.")));
+          "Invalid content provided by the connector \"" + getConnector().id
+          + "\": Invalid JSON type. Expected is a sub-type of XyzResponse.")));
     } catch (JsonParseException e) {
-      logger.error(marker, "Error in the provided content", e);
-      callback.handle(Future.failedFuture(new HttpException(BAD_GATEWAY, "Invalid content provided by the connector: Invalid JSON string. "
-          + "Error at line " + e.getLocation().getLineNr() + ", column " + e.getLocation().getColumnNr() + ".")));
+      logger.error(marker, "Error in the provided content from connector \"{}\"", getConnector().id, e);
+      callback.handle(Future.failedFuture(new HttpException(BAD_GATEWAY, "Invalid content provided by the connector \"" + getConnector().id
```

Comment:
This kind of detail should not be part of the actual error response going to the client.